Yoast/wordpress-seo
# DUPP-670 Refactor Input_Helper and Estimated_Reading_Time_Conditional

## Context


* We want to replace all `filter_input` calls in our codebase.
* This PR also removes two places of the `Estimated_Reading_Time_Conditional` class where it was not used anymore. The PR where the instances were added was [this PR](https://github.com/Yoast/wordpress-seo/commit/3921caecf76bfd74101ce35b55052998c9697a31) and the PR where the usage was removed but the class initialize was still left is [this PR](https://github.com/Yoast/wordpress-seo/commit/f0b05c9ea83aa8423a4043019505e73d27ca61c7).

## Summary


This PR can be summarized in the following changelog entry:

* Remove the `Input_Helper` class and refactor the places where it was used.
* Remove instances of `Estimated_Reading_Time_Conditional` where it was not used.

## Relevant technical choices:

* This PR also removes two instances of the `Estimated_Reading_Time_Conditional` in the Elementor metabox and the normal metabox because it was not used anymore.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

**Test the normal metabox**
* Install and activate Yoast SEO.
* Go to a post editor screen.
* Verify that the Yoast SEO metabox still renders.
* Now change some settings in the metabox and save.
* Verify whether the changes are applied correctly after a page refresh.

**Test the Elementor metabox**
* Install and activate Elementor.
* Go to a post editor and edit it with Elementor.
* In the Elementor editor, click on the hamburger menu at the top left of the screen.
* Click on Yoast SEO under More and verify whether the metabox still renders.
* Now change some settings in the metabox and save.
* Verify whether the changes are applied correctly after a page refresh.

**Test the `Estimated_Reading_Time_Conditional`**
* Go to a post editor screen.
* Open the HTML content of the post editor screen and search for `yoast_wpseo_estimated-reading-time-minutes`.
* Verify whether an `<input type="hidden">` field pops up (something like `<input type="hidden" id="yoast_wpseo_estimated-reading-time-minutes" name="yoast_wpseo_estimated-reading-time-minutes" value="1">`).

* Now go to an Elementor editor screen and once again open the HTML content of the page.
* Search for `yoast_wpseo_estimated-reading-time-minutes`.
* Verify whether an `<input type="hidden">` field pops up (something like `<input type="hidden" id="yoast_wpseo_estimated-reading-time-minutes" name="yoast_wpseo_estimated-reading-time-minutes" value="1">`).

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #
